doc/sphinx/proof-engine/tactics.rst

```diff
@@ -611,26 +625,29 @@ Applying theorems
    .. exn:: No such assumption.
       :undocumented:
 
-   .. tacv:: eassumption
-      :name: eassumption
+   .. tacn:: eassumption
 
-      This tactic behaves like :tacn:`assumption` but is able to handle
-      goals with existential variables.
+      Behaves like :tacn:`assumption` but is able to handle
+      goals and hypotheses with existential variables.
 
-.. tacn:: refine @term
+.. tacn:: {? simple } {? notypeclasses } refine @one_term
    :name: refine
 
-   This tactic applies to any goal. It behaves like :tacn:`exact` with a big
-   difference: the user can leave some holes (denoted by ``_``
-   or :n:`(_ : @type)`) in the term. :tacn:`refine` will generate as many
+   Behaves like :tacn:`exact` but allows holes (denoted by ``_``
+   or :n:`(_ : @type)`) in :n:`@one_term`. :tacn:`refine` generates as many
    subgoals as there are remaining holes in the elaborated term. The type
```

Comment:
```suggestion
   subgoals as there are remaining holes in the elaborated term. The types
```

Reply:
See below

Reply:
@Alizter Note that you can do suggestions spanning multiple lines by first selecting the few lines on which you want to comment.